src/Infusionsoft/Infusionsoft.php

```diff
@@ -226,7 +230,7 @@ public function requestAccessToken($code)
 
         $client = $this->getHttpClient();
 
-        $tokenInfo = $client->request($this->tokenUri, $params, array(), 'POST');
+        $tokenInfo = $client->request('POST', $this->tokenUri, ['body' => $params] );
```

Comment:
It now throws a new fatal error.

```php
Fatal error: Uncaught exception 'InvalidArgumentException' with message 'Invalid resource type: array' in /infusionsoft-php/vendor/guzzlehttp/psr7/src/functions.php:113 Stack trace: #0 /infusionsoft-php/vendor/guzzlehttp/psr7/src/Request.php(62): GuzzleHttp\Psr7\stream_for(Array) #1 /infusionsoft-php/src/Infusionsoft/Http/GuzzleHttpClient.php(67): GuzzleHttp\Psr7\Request->__construct('POST', 'https://api.inf...', Array, Array) #2 /infusionsoft-php/src/Infusionsoft/Infusionsoft.php(233): Infusionsoft\Http\GuzzleHttpClient->request('POST', 'https://api.inf...', Array) #3 /infusionsoft-php/samples/addWithDupCheck.php(27): Infusionsoft\Infusionsoft->requestAccessToken('w7rysy85yvgcy7n...') #4 {main} thrown in /infusionsoft-php/vendor/guzzlehttp/psr7/src/functions.php on line 113
```

In looking at the fucntions.php , it doesn't allow for a type array, only "string", "resource","object", or "NULL"